packages/tests/src/constants.ts

```diff
@@ -50,7 +50,8 @@ export const TEST_STRING = [
   { description: "JSON", value: '{"user":"admin","password":"123456"}' },
   { description: "shell command", value: "`rm -rf /`" },
   { description: "escaped characters", value: "\\n\\t\\0" },
-  { description: "unicode special characters", value: "\u202Ereverse" }
+  { description: "unicode special characters", value: "\u202Ereverse" },
+  { description: "emoji", value: "🤫 🤥 😶 😶‍🌫️ 😐 😑 😬 🫨 🫠 🙄 😯 😦 😧 😮" }
```

Comment:
If I understand correctly, these are testing the `utf8ToByte` utils functions which are here are helpers and not expected to be used by any serious project.

using fast-check to generate arbitrary byte arrays as payload may be a better approach @fbarbu15

Reply:
Indeed for some cases, those string are converted to bytes when sending the message, and then upon assertion converted back from bytes to string. So in those cases you are right, we are actually checking the utils and using fast-check indeed looks a better approach.

But there are cases when we use those string as contentTopic, and in that case I think it's more relevant to have those kind of hardcoded edge cases. Ex of such message retrieved from nwaku:

  {
    payload: '8J+kqyDwn6SlIPCfmLYg8J+YtuKAjfCfjKvvuI8g8J+YkCDwn5iRIPCfmKwg8J+rqCDwn6ugIPCfmYQg8J+YryDwn5imIPCfmKcg8J+Yrg==',
    contentTopic: '🤫 🤥 😶 😶‍🌫️ 😐 😑 😬 🫨 🫠 🙄 😯 😦 😧 😮',
    version: 0,
    timestamp: 1698130469857000000,
    ephemeral: false
  }